framework-plugins/lisk-framework-http-api-plugin/test/functional/blocks.spec.ts

```diff
@@ -11,26 +11,27 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
-import { Application } from 'lisk-framework';
+import { testing } from 'lisk-framework';
 import axios from 'axios';
 import {
 	callNetwork,
-	createApplication,
-	closeApplication,
+	createApplicationEnv,
+	closeApplicationEnv,
 	getURL,
 	waitNBlocks,
 } from './utils/application';
 
 describe('Blocks endpoints', () => {
-	let app: Application;
+	let appEnv: testing.ApplicationEnv;
 
 	beforeAll(async () => {
-		app = await createApplication('blocks_http_functional');
-		await waitNBlocks(app, 1);
+		appEnv = createApplicationEnv('blocks_http_functional');
+		await appEnv.startApplication();
+		await waitNBlocks(appEnv.application, 1);
```

Comment:
It should be updated to use

```suggestion
		await appEnv.waitNBlocks(1);
```